packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,40 +1,54 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-item-background-color: defines the background color for the component.
+ * @prop --calcite-accordion-item-border-color: defines the border color for the component.
+ * @prop --calcite-accordion-item-description-text-color: defines the text color of the description in the component. Falls back to `--calcite-accordion-item-text-color`.
```

Comment:
For the time being, can we omit specifying which CSS prop it will fall back to? I think we could do so since it's an implementation detail.